DEV.md

```diff
@@ -0,0 +1,232 @@
++ Top Level Design
+
+The Eclipse Transformer comprises:
+
+  * File update actions;
+  * Which are property file driven;
+  * Composed with container actions;
+  * And invoked using java and command line APIs.
+
+Further:
+
+  * The core operation is to create a transformed copy of a root input file:
+    ** The core inputs are a target root input file, a target root output file, and a set of processing options.
```

Comment:
For nested bullets you need 4 spaces indent from the outer level bullet.

Reply:
Corrected by refresh of the files.